partcad/src/partcad/cache_shape.py

```
@@ -0,0 +1,130 @@
+                results[key] = False
+            elif (
+                key_is_cached_in_files
+                and user_config.cache_memory_double_cache_max_entry_size > 0
+                and data_len > user_config.cache_memory_double_cache_max_entry_size
+            ):
+                # The object is bigger than what we want to store in both caches
+                results[key] = False
+            else:
+                results[key] = True
+
+            return results
```

Ruff (0.8.2)

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)